README.md

```diff
@@ -1,3 +1,5 @@
+This project is a component of the [Operator Framework](https://github.com/operator-framework), an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the [introduction blog post](https://coreos.com/blog/introducing-operator-framework).
```

Comment:
We should give credit to the [upstream project, Lostromós](https://github.com/wpengine/lostromos) that the `coreos/sao` image is built from.

Reply:
Where do you want to do this?

Reply:
Added below. Wasnt sure if you wanted to update the quay repo description too

Reply:
I've already updated https://quay.io/repository/coreos/sao.